src/lib/storage/chunk.cpp

```diff
@@ -1,31 +1,65 @@
 #include "chunk.hpp"
 
-#include "abstract_segment.hpp"
 #include "utils/assert.hpp"
 
 namespace opossum {
 
-void Chunk::add_segment(const std::shared_ptr<AbstractSegment> segment) {
+void Chunk::add_segment(const std::shared_ptr<AbstractSegment> column) {
   // Implementation goes here
```

Comment:
Not sure whether we should keep the original signature (not rename segment to column). If we want to stick with column we should also rename it in the .hpp though.

We should also probably remove the comments stating "Implementation goes here"

Reply:
Lets change as few as possible.